src/tests/evaluation.test.ts

```
@@ -0,0 +1,128 @@
+      expect(report).toContain('test-2');
+    });
+
+    it('should include duration in the report', async () => {
+      const report = await evaluationService.evaluate(testCases);
+      expect(report).toMatch(/Duration: \d+\.\d+s/);
+    });
+
+    it('should handle test execution errors gracefully', async () => {
+      // Mock the executeTest method to throw an error
+      jest
+        .spyOn(evaluationService as any, 'executeTest')
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)